src/Core/src/Response.php

```diff
@@ -25,20 +25,21 @@
  */
 class Response
 {
-    private $response;
+    private $httpResponse;
```

Comment:
We currently have `Result::$response` and `Response::$response`. This makes it a little bit more clear.